copy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+container_id=$1
+dir=""
+copy_to_host() {
```

Comment:
copy.sh -> compile.sh ， include serveral step: 
1. docker pull kmesh compile-tool image.
2. compile kmesh in compile-tool image.
3. copy compile result to .out dir

docker build